src/authentication/service/auth.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { User } from '../entities/user.entity';
+import { Repository } from 'typeorm';
+import { ReqUser } from '../dto/user.dto/users.dto';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    @InjectRepository(User) private userRepository: Repository<User>,
+  ) {}
+  async getUserInfo(request: Request): Promise<ReqUser> {
+    const User: User[] = await this.userRepository.findBy({
+      userIdToken: request['user'].user_id,
```

Comment:
Please avoid this syntax, what is `userIdToken`? Why it's equal to userId

Reply:
I've created a decorator to use user information where needed, as you suggested below.